config/locales/en.yml

```diff
@@ -800,6 +821,7 @@ en:
   label_information_plural: "Information"
   label_integer: "Integer"
   label_internal: "Internal"
+  label_invite_user: "Invite user"
```

Comment:
No, invite user will hit the core as well, and does so in the next release. Niels asked me to name it this way.